### PR TITLE
Convert `web/api/element` to markdown for Spanish (es)

### PR DESCRIPTION
**Pull request by jakul**
Part of the larger 100% Markdown [project](https://github.com/mdn/translated-content/issues/7486)

### CODE REVIEW

**Comment by jakul** [comment]
@Graywolf9 FYI

File: files/es/web/api/element/clientleft/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+slug: Web/API/Element/clientLeft
+tags:
+  - Referencia_DOM_de_Gecko
+translation_of: Web/API/Element/clientLeft
+---
+{{ APIRef }}
+
+## Resumen
+
+El ancho del borde izquierdo de un elemento en pixel. Incluye el ancho de la barra de desplazamiento vertical, si la dirección del texto es*rtl* (derecha a izquierda) y si hay un desbordamiento que provoque la aparición de la barra en el lado izquierdo. `clientLeft` NO incluye*margin-left* o padding-left*. `clientLeft` es de sólo lectura.*
```
Comment (Graywolf9) [inline files/es/web/api/element/clientleft/index.md:12]:
```suggestion
El ancho del borde izquierdo de un elemento en pixel. Incluye el ancho de la barra de desplazamiento vertical, si la dirección del texto es*rtl* (derecha a izquierda) y si hay un desbordamiento que provoque la aparición de la barra en el lado izquierdo. `clientLeft` NO incluye *margin-left* o *padding-left*. `clientLeft` es de sólo lectura.
```

File: files/es/web/api/element/clientleft/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<img alt="Image:clientLeft.png">
+
+<img alt="La barra de desplazamiento vertical en el lado izquierdo, aparece cuando la propiedad layout.scrollbar.side tiene el valor 1 o 3">
+
+Cuando la propiedad [_layout.scrollbar.side_](http://kb.mozillazine.org/Layout.scrollbar.side)se establece a 1 o a 3 y la dirección de texto está puesta a RTL, **entonces la barra de desplazamiento se pone a la izquierda** y esto afecta a la forma en que se calcula clientLeft.
```
Comment (Graywolf9) [inline files/es/web/api/element/clientleft/index.md:40]:
````suggestion
```html hidden
<div id="offsetContainer" style="margin: 26px 0px; background-color: rgb(255, 255, 204); border: 4px dashed black; color: black; position: absolute; left: 260px;">
<div id="idDiv" style="margin: 24px 29px; border: 24px black solid; padding: 0px 28px; width: 199px; height: 102px; overflow: auto; background-color: white; font-size: 13px!important; font-family: Arial, sans-serif;">
<p id="PaddingTopLabel" style="text-align: center; font-style: italic; font-weight: bold; font-size: 13px!important; font-family: Arial, sans-serif; margin: 0px;">padding-top</p>

<p>Suave, individualista y muy leal, los gatos de Birmania tienen un carácter entre los persas y los siameses. Si le gustan los gatos que no son agresivos, disfrutan estar entre los humanos y son generalmente tranquilos, esta es la raza para usted.</p>

<p><span style="float: right;"><img alt="Image:BirmanCat.jpg"></span>Todos los Birmanos tienen pintas y coloración obscura en la cara, orejas, patas y cola.</p>

<p>La imagen del gato y el texto vienen de <a class="external" href="http://www.best-cat-art.com/">www.best-cat-art.com</a></p>

<p id="PaddingBottomLabel" style="text-align: center; font-style: italic; font-weight: bold; font-size: 13px!important; font-family: Arial, sans-serif; margin: 0px;">padding-bottom</p>
</div>
<strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: -32px; position: absolute; top: 85px;">Left</strong><strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 170px; position: absolute; top: -24px;">Top</strong><strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 370px; position: absolute; top: 85px;">Right</strong><strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 164px; position: absolute; top: 203px;">Bottom</strong></div>
```

{{EmbedLiveSample("",400,300)}}

<img alt="Image:clientLeft.png">

<img alt="La barra de desplazamiento vertical en el lado izquierdo, aparece cuando la propiedad layout.scrollbar.side tiene el valor 1 o 3">

Cuando la propiedad [_layout.scrollbar.side_](http://kb.mozillazine.org/Layout.scrollbar.side)se establece a 1 o a 3 y la dirección de texto está puesta a RTL, **entonces la barra de desplazamiento se pone a la izquierda** y esto afecta a la forma en que se calcula clientLeft.
````

File: files/es/web/api/element/clienttop/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+<img alt="Image:clientTop.png">
```
Comment (Graywolf9) [inline files/es/web/api/element/clienttop/index.md:36]:
````suggestion
```html hidden
<div id="offsetContainer" style="margin: 26px 0px; background-color: rgb(255, 255, 204); border: 4px dashed black; color: black; position: absolute; left: 260px;">
<div id="idDiv" style="margin: 24px 29px; border: 24px black solid; padding: 0px 28px; width: 199px; height: 102px; overflow: auto; background-color: white; font-size: 13px!important; font-family: Arial, sans-serif;">
<p id="PaddingTopLabel" style="text-align: center; font-style: italic; font-weight: bold; font-size: 13px!important; font-family: Arial, sans-serif; margin: 0px;">padding-top</p>

<p>Suave, individualista y muy leal, los gatos de Birmania tienen un carácter entre los persas y los siameses. Si le gustan los gatos que no son agresivos, disfrutan estar entre los humanos y son generalmente tranquilos, esta es la raza para usted.</p>

<p><span style="float: right;"><img alt="Image:BirmanCat.jpg"></span>Todos los Birmanos tienen pintas y coloración obscura en la cara, orejas, patas y cola.</p>

<p>La imagen del gato y el texto vienen de <a class="external" href="http://www.best-cat-art.com/">www.best-cat-art.com</a></p>

<p id="PaddingBottomLabel" style="text-align: center; font-style: italic; font-weight: bold; font-size: 13px!important; font-family: Arial, sans-serif; margin: 0px;">padding-bottom</p>
</div>
<strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: -32px; position: absolute; top: 85px;">Left</strong><strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 170px; position: absolute; top: -24px;">Top</strong><strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 370px; position: absolute; top: 85px;">Right</strong><strong style="color: blue; font-family: Arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 164px; position: absolute; top: 203px;">Bottom</strong></div>
```

{{EmbedLiveSample("",400,300)}}

<img alt="Image:clientTop.png">
````

File: files/es/web/api/element/innerhtml/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+```js
+const content = element.innerHTML;
+
+element.innerHTML = htmlString;
+```
+
+### Valor
+
+La variable constante `content` contiene un [DOMString](/es/docs/Web/API/DOMString) que contiene el código HTML serializado describiendo todos los descendientes de `element`. Cuando se establece el valor de innerHTML, **se eliminan todos los descendientes de `element`, analiza la cadena `htmString` y asigna los nodos resultantes como descendientes de `element`.
```
Comment (Graywolf9) [inline files/es/web/api/element/innerhtml/index.md:34]:
```suggestion
La variable constante `content` contiene un [DOMString](/es/docs/Web/API/DOMString) que contiene el código HTML serializado describiendo todos los descendientes de `element`. Cuando se establece el valor de innerHTML, **se eliminan todos los descendientes de `element`**, analiza la cadena `htmString` y asigna los nodos resultantes como descendientes de `element`.
```

File: files/es/web/api/element/scrollheight/index.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+padding-top
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+padding-bottom
+
+**Left** **Top** **Right** **Bottom** _margin-top_ _margin-bottom_ _border-top_ _border-bottom_
```
Comment (Graywolf9) [inline files/es/web/api/element/scrollheight/index.md:36]:
````suggestion
```html hidden
<div id="offsetContainer" style="margin: 40px 50px 50px; background-color: rgb(255, 255, 204); border: 4px dashed black; color: black; position: relative; display: inline-block;">
<div id="idDiv" style="margin: 24px 29px; border: 24px black solid; padding: 0px 28px; width: 199px; height: 102px; overflow: auto; background-color: white; font-size: 13px!important; font-family: Arial, sans-serif;">
<p id="PaddingTopLabel" style="text-align: center; font-style: italic; font-weight: bold; font-size: 13px!important; font-family: Arial, sans-serif; margin: 0px;">padding-top</p>

<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>

<p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>

<p id="PaddingBottomLabel" style="text-align: center; font-style: italic; font-weight: bold; font-size: 13px!important; font-family: Arial, sans-serif; margin: 0px;">padding-bottom</p>
</div>
<strong>Left</strong> <strong>Top</strong> <strong style="color: blue; font-family: arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 370px; position: absolute; top: 85px;">Right</strong> <strong style="color: blue; font-family: arial,sans-serif; font-size: 13px!important; font-weight: bold; left: 164px; position: absolute; top: 203px;">Bottom</strong> <em>margin-top</em> <em>margin-bottom</em> <em>border-top</em> <em>border-bottom</em></div>
```

{{EmbedLiveSample("",400,350)}}
````

File: files/es/web/api/element/scrolltopmax/index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+## Sintaxis
+
+```js
+var pxl = elt.scrollTopMax;
+```
+
+## Especificaciones
+
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/api/element/scrolltopmax/index.md:26]:
```suggestion
Esta propiedad no es parte de ninguna especificación.
```

File: files/es/web/api/element/scrolltop/index.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+La propiedad `Element.scrollTop` obtiene o establece el número de pixels que el contenido de un elemento ha sido desplazado (enrollado) hacia arriba. El `scrollTop` de un elemento es una medida de la distancia desde el límite superior de un elemento al límite superior de su contenido visible. Cuando un elemento no genera una barra de scroll vertical, el valor de su `scrollTop` es establecido por defecto a `0`.
+
+## Sintaxis
+
+```js
+// Obtiene el número de pixels desplazados
+var  intElemScrollTop = someElement.scrollTop;
+```
+
+_Después de ejecutar este código, `intElemScrollTop` es un entero que corresponde al número de pixesl que el contenido del {{domxref("element")}} ha sido desplazado hacia arriba.
```
Comment (Graywolf9) [inline files/es/web/api/element/scrolltop/index.md:17]:
```suggestion
Después de ejecutar este código, `intElemScrollTop` es un entero que corresponde al número de pixesl que el contenido del {{domxref("element")}} ha sido desplazado hacia arriba.
```

File: files/es/web/api/element/scroll_event/index.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+---
+title: GlobalEventHandlers.onscroll
+slug: Web/API/Element/scroll_event
+translation_of: Web/API/GlobalEventHandlers/onscroll
+original_slug: Web/API/GlobalEventHandlers/onscroll
+---
+}}
```
Comment (Graywolf9) [inline files/es/web/api/element/scroll_event/index.md:7]:
```suggestion
{{ApiRef("HTML DOM")}}
```

File: files/es/web/api/element/setcapture/index.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+  - Element
+  - Método(2)
+  - Referencia
+translation_of: Web/API/Element/setCapture
+---
+{{ APIRef }}
+
+Llame a este método durante el manejo de un evento mousedown para redirigir todos los eventos de mouse a este elemento hasta que se suelte el botón del mouse o se llame {{domxref ( "document.releaseCapture ()")}}.
+
+## Sintasix
```
Comment (Graywolf9) [inline files/es/web/api/element/setcapture/index.md:16]:
```suggestion
## Sintaxis
```

File: files/es/web/api/element/wheel_event/index.md [hunk 10]
```diff
@@ -7,32 +7,20 @@ browser-compat: api.Element.wheel_event
...
+## Propiedades
+
+- Burbujas
+  - : Sí
+- Cancelable
+  - : Sí
+- Interfaz
+  - : {{domxref("WheelEvent")}}
+- Propiedades del manejador de eventos
+  - : {{domxref("GlobalEventHandlers.onwheel", "onwheel")}}
```
Comment (Graywolf9) [inline files/es/web/api/element/wheel_event/index.md:23]: I'm not sure why did you change the format

```suggestion
| Burbujas                             | Sí                                                                                                   |
|--------------------------------------|------------------------------------------------------------------------------------------------------|
| Cancelable                           | Sí                                                                                                   |
| Interfaz                             | [`WheelEvent`](https://developer.mozilla.org/es/docs/Web/API/WheelEvent)                             |
| Propiedades del manejador de eventos | [`onwheel`](https://developer.mozilla.org/es/docs/conflicting/Web/API/Element/wheel_event "onwheel") |
```

File: files/es/web/api/element/index.md [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+| `setAttributeNS( namespace, name, value )` Establece el valor del atributo con el nombre y nombre de espacio especificado desde el nodo actual.                                | -                                                                                                                             | [Todos](/es/DOM "es/DOM")                                |
+| `setAttributeNode( name, attrNode )` Establece la representación del nodo del atributo nombrado desde el nodo actual.                                                          | -                                                                                                                             | [Todos](/es/DOM "es/DOM")                                |
+| `setAttributeNodeNS( namespace, name, attrNode )` Establece la representación del nodo del atributo con el nombre y nombre de espacio especificado desde el nodo actual.       | -                                                                                                                             | [Todos](/es/DOM "es/DOM")                                |
```
Comment (Graywolf9) [inline files/es/web/api/element/index.md:98]:
```suggestion
| Nombre y Descripción                                                                                                                                                                                                                                                                                                                                                           | Devuelve                                                            | Disponible                                     |
| ---                                                                                                                                                                                                                                                                                                                                                                            | ---                                                                 | ---                                            |
| [`addEventListener`](/es/docs/DOM/element.addEventListener) ( [type](/es/docs/Web/JavaScript/Reference/Global_Objects/String), [handler](/es/JavaScript/Reference/Global_Objects/Function), [bubble](/es/JavaScript/Reference/Global_Objects/Boolean)) <br>Registra un controlador de evento para un tipo de evento específico en un elemento.                                   | -                                                                   | [Todos](/es/docs/Web/API/Event)                |
| [`appendChild`](/es/docs/Web/API/Node/appendChild)( [appendedNode](/es/docs/DOM/Node) )  <br>Inserta un nodo así como el último nodo hijo de este elemento.                                                                                                                                                                                                                      | [Node](/es/docs/DOM/Node)                                           | [Todos](/es/docs/DOM)                          |
| [`blur`](/es/docs/DOM/element.blur)()  <br>Quita el foco del teclado del elemento actual.                                                                                                                                                                                                                                                                                        | -                                                                   | para [HTML](/es/docs/Web/HTML), [XUL](/es/XUL) |
| [`click`](/es/docs/DOM/element.click)()  <br>Simula un clic sobre el elemento actual.                                                                                                                                                                                                                                                                                            | -                                                                   | para [HTML](/es/docs/Web/HTML), [XUL](/es/XUL) |
| [`cloneNode`](/es/docs/Web/API/Node/cloneNode)( [deep](/es/JavaScript/Reference/Global_Objects/Boolean))  <br>Hace copia de un nudo, y opcionalmente, de todo sus contenidos                                                                                                                                                                                                     | [Node](/es/docs/DOM/Node)                                           | para [Todos](/es/docs/DOM)                     |
| [`dispatchEvent`](/es/docs/Web/API/EventTarget/dispatchEvent)( [event](/es/docs/Web/API/Event) )  <br>Envía un evento a este nodo en el DOM.                                                                                                                                                                                                                                     | [Boolean](/es/Core_JavaScript_1.5_Reference/Global_Objects/Boolean) | [Todos](/es/docs/DOM)                          |
| [`getAttribute`](/es/docs/Web/API/Element/getAttribute)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Devuelve el valor de un atributo nombrado desde el nodo actual.                                                                                                                                                                                  | [Object](/es/Core_JavaScript_1.5_Reference/Global_Objects/Object)   | [Todos](/es/docs/DOM)                          |
| [`getAttributeNS`](/es/docs/Web/API/Element/getAttributeNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Devuelve el valor del atributo con el nombre especificado, desde el nodo actual.                                                                                       | [Object](/es/Core_JavaScript_1.5_Reference/Global_Objects/Object)   | [Todos](/es/docs/DOM)                          |
| [`getAttributeNode`](/es/docs/Web/API/Element/getAttributeNode)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Devuelve la representación del nodo del atributo nombrado desde el nodo actual.                                                                                                                                                          | [Attr](/es/docs/DOM/Attr)                                           | [Todos](/es/docs/DOM)                          |
| [`getAttributeNodeNS`](/es/docs/Web/API/Element/getAttributeNodeNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Devuelve la representación del nodo del atributo con el nombre especificado, desde el nodo actual.                                                             | [Attr](/es/docs/DOM/Attr)                                           | [Todos](/es/docs/DOM)                          |
| Nombre y Descripción                                                                                                                                                                                                                                                                                                                                                           | Devuelve                                                            | Disponibilidad                                 |
| [`getElementsByTagName`](/es/docs/Web/API/Element/getElementsByTagName)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Devuelve un conjunto de todos los elementos descendentes, de un nombre de etiqueta particular, desde el elemento actual.                                                                                                         | [NodeSet](/es/docs/DOM/NodeSet)                                     | [Todos](/es/docs/DOM)                          |
| [`getElementsByTagNameNS`](/es/docs/Web/API/Element/getElementsByTagNameNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Devuelve un conjunto de todos los elementos descendentes, de un nombre de etiqueta y espacio particular, desde el elemento actual.                     | [NodeSet](/es/docs/DOM/NodeSet)                                     | [Todos](/es/docs/DOM)                          |
| [`hasAttribute`](/es/docs/Web/API/Element/hasAttribute)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Verifica si el elemento tiene el atributo especificado o no.                                                                                                                                                                                     | [Boolean](/es/Core_JavaScript_1.5_Reference/Global_Objects/Boolean) | [Todos](/es/docs/DOM)                          |
| [`hasAttributeNS`](/es/docs/Web/API/Element/hasAttributeNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Verifica si el elemento tiene el atributo especificado, en el nombre de espacio especificado o no.                                                                     | [Boolean](/es/Core_JavaScript_1.5_Reference/Global_Objects/Boolean) | [Todos](/es/docs/DOM)                          |
| [`hasAttributes`](/es/docs/Web/API/Element/hasAttributes)()  <br>Verifica si el elemento tiene o no algún atributo.                                                                                                                                                                                                                                                              | [Boolean](/es/Core_JavaScript_1.5_Reference/Global_Objects/Boolean) | [Todos](/es/docs/DOM)                          |
| [`hasChildNodes`](/es/docs/Web/API/Node/hasChildNodes)()  <br>Verifica si el elemento tiene nodos hijos o no.                                                                                                                                                                                                                                                                    | [Boolean](/es/Core_JavaScript_1.5_Reference/Global_Objects/Boolean) | [Todos](/es/docs/DOM)                          |
| [`insertBefore`](/es/docs/Web/API/Node/insertBefore)( [insertedNode](/es/docs/DOM/Node), [adjacentNode](/es/docs/DOM/Node) )  <br>Inserta el primer nodo antes que el segundo, Nodo hijo en el DOM.                                                                                                                                                                              | [Node](/es/docs/DOM/Node)                                           | [Todos](/es/docs/DOM)                          |
| [`normalize`](/es/docs/Web/API/Node/normalize)()  <br>Limpia todos los nodos de texto debajo de este elemento (une lo adyacente, quita lo vacío).                                                                                                                                                                                                                                | -                                                                   | [Todos](/es/docs/DOM)                          |
| [`removeAttribute`](/es/docs/Web/API/Element/removeAttribute)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Quita el atributo nombrado desde el nodo actual.                                                                                                                                                                                           | -                                                                   | [All](/es/docs/DOM)                            |
| [`removeAttributeNS`](/es/docs/Web/API/Element/removeAttributeNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Quita el atributo con el nombre y nombre de espacio especificado desde el nodo actual.                                                                           | -                                                                   | [Todos](/es/docs/DOM)                          |
| Nombre y Descripción                                                                                                                                                                                                                                                                                                                                                           | Devuelve                                                            | Disponibilidad                                 |
| [`removeAttributeNode`](/es/docs/Web/API/Element/removeAttributeNode)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String) )  <br>Quita la representación del nodo del atributo nombrado desde el nodo actual.                                                                                                                                                       | -                                                                   | [Todos](/es/docs/DOM)                          |
| [`removeChild`](/es/docs/Web/API/Node/removeChild)( [removedNode](/es/docs/DOM/Node) )  <br>Quita el nodo hijo desde el elemento actual.                                                                                                                                                                                                                                         | [Node](/es/docs/DOM/Node)                                           | [Todos](/es/docs/DOM)                          |
| [`removeEventListener`](/es/docs/Web/API/Element/removeEventListener)( [type](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [handler](/es/Core_JavaScript_1.5_Reference/Global_Objects/Function) )  <br>Quita un oyente de evento desde el elemento.                                                                                                                 | -                                                                   | [Todos](/es/docs/Web/API/Event)                |
| [`replaceChild`](/es/docs/Web/API/Node/replaceChild)( [insertedNode](/es/docs/DOM/Node), [replacedNode](/es/docs/DOM/Node) )  <br>Reemplaza un nodo hijo en el elemento actual con otro.                                                                                                                                                                                         | [Node](/es/docs/DOM/Node)                                           | [Todos](/es/docs/DOM)                          |
| [`scrollIntoView`](/es/docs/Web/API/Element/scrollIntoView)( [alignWithTop](/es/Core_JavaScript_1.5_Reference/Global_Objects/Boolean) )  <br>Recorre la página hasta que el elemento se obtiene en la vista.                                                                                                                                                                     | -                                                                   | [HTML](/es/docs/Web/HTML)                      |
| [`setAttribute`](/es/docs/Web/API/Element/setAttribute)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [value](/es/Core_JavaScript_1.5_Reference/Global_Objects/Object) )  <br>Establece el valor de un atributo nombrado desde el nodo actual.                                                                                                               | -                                                                   | [Todos](/es/docs/DOM)                          |
| [`setAttributeNS`](/es/docs/Web/API/Element/setAttributeNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [value](/es/Core_JavaScript_1.5_Reference/Global_Objects/Object) )  <br>Establece el valor del atributo con el nombre y nombre de espacio especificado desde el nodo actual. | -                                                                   | [Todos](/es/docs/DOM)                          |
| [`setAttributeNode`](/es/docs/Web/API/Element/setAttributeNode)( [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [attrNode](/es/docs/DOM/Attr) )  <br>Establece la representación del nodo del atributo nombrado desde el nodo actual.                                                                                                                          | -                                                                   | [Todos](/es/docs/DOM)                          |
| [`setAttributeNodeNS`](/es/docs/Web/API/Element/setAttributeNodeNS)( [namespace](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [name](/es/Core_JavaScript_1.5_Reference/Global_Objects/String), [attrNode](/es/docs/DOM/Attr) )  <br>Establece la representación del nodo del atributo con el nombre y nombre de espacio especificado desde el nodo actual.          | -                                                                   | [Todos](/es/docs/DOM)                          |
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for your awesome job @jakul !, I took the time to fix some errors
